versions.txt

```diff
@@ -1 +1 @@
-1.0.1
\ No newline at end of file
+1.0.2
```

Comment:
@MariaMaximova, considering you are mergin into dev branch, I would suggest you not changing version here. You can change it when you will create an MR to main branch.

Reply:
Version was reverted back to [v1.0.1](https://github.com/MariaMaximova/convert_temp/pull/9/commits/1ab8b94e86226a891571c2a8960d6bb6b781e07d#diff-eb8281dc5868dec21668321219be72bacb9baa0acf7e43ac159d5a53e9d44abc)